android/app/build.gradle

```diff
@@ -54,7 +54,7 @@ android {
 
     defaultConfig {
         applicationId "com.jonjomckay.fritter"
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
Is there a reason this has to be bumped to 19? I do still have pings from users on 16, so I'd like to keep Fritter working for them if possible!